setup.py

```diff
@@ -44,6 +44,7 @@
         "pandas",
         "plotly>=3.6,<4.*",
         "requests",
+        "spiceypy>=2.2.0",
```

Comment:
Can we add this to `extras_require["dev"]` instead?

Reply:
Ok